sync/src/synchronizer/get_blocks_process.rs

```diff
@@ -51,14 +51,6 @@ impl<'a> GetBlocksProcess<'a> {
                 continue;
             }
 
-            if self.nc.send_paused() {
```

Comment:
How can we know that the send buffer is full now? Will `self.nc.send_message_to` return an error?

cc @driftluo

Reply:
An error will be returned on the service handle, and then the overloaded session will be forcibly closed

Reply:
Will this loop return early so we don't waste time to fetch the block from database and push it to the queue.

Reply:
The framework cannot control this behavior. If the session is killed here because exceed the upper limit of the buffer, sending any data is an invalid operation and will be discarded immediately, but it cannot interrupt the loop code